site/src/examples/card/LinkCardExample.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactElement } from "react";
+import { LinkCard } from "@salt-ds/lab";
+
+export const LinkCardExample = (): ReactElement => {
+  return (
+    <LinkCard
+      href="https://www.saltdesignsystem.com"
```

Comment:
```suggestion
      href="#"
```
clicking is currently redirecting to the homepage